mobify/progressive-web-scaffold
# Documentation: Previewing on an Android device

This PR extends the note about previewing with SSL. It also re-arranges the `README` docs in light of the recent folder moves (native/web).

 **JIRA**: N/A
 **Linked PRs**: N/A

## Changes
- Adds documentation outlining how to preview on an Android device
- Creates [README.md](/README.md)
- Moves some content from [web/README.md](/web/README.md) to [README.md](/README.md)

## How to test-drive this PR
- Just read the doc updates
- Make sure the portions moved to [README.md](/README.md) make sense to be there
